app/assets/javascripts/Components/Modals/graders_distribution_modal.jsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import Modal from "react-modal";
+import PropTypes from "prop-types";
+
+export class GraderDistributionModal extends React.Component {
+  static defaultProps = {
+    override: false,
+  };
+
+  constructor(props) {
+    super(props);
+    this.input = React.createRef();
+  }
+
+  componentDidMount() {
+    Modal.setAppElement("body");
+  }
+
+  onSubmit = event => {
+    event.preventDefault();
+    const weightings = {};
+    const form = new FormData(this.input.current);
+    form.forEach((key, value) => {
+      weightings[value] = key;
+    });
+    this.props.onSubmit(weightings);
+  };
+
+  renderGraderRow = grader => {
+    return (
+      <div className="flex-row-expand" key={grader.user_name}>
+        <label htmlFor={`input-${grader.user_name}`} className="modal-inline-label">
+          {grader.user_name}
+        </label>
+        <input
+          id={`input-${grader.user_name}`}
+          type="number"
+          step="0.01"
+          min="0"
+          max="100"
```

Comment:
No need for a `max` attribute (since these weights aren't necessarily percentages)